config/dev.exs

```diff
@@ -57,7 +57,6 @@ config :bookmark, BookmarkWeb.Endpoint,
 config :bookmark, BookmarkWeb.Endpoint,
   live_reload: [
     patterns: [
-      ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
```

Comment:
Why it was removed?

Reply:
In order to run locally, I removed this because the archive goes in this folder.